cogs5e/models/character.py

```diff
@@ -156,15 +160,18 @@ def from_dict(cls, d):
         return cls(**d)
 
     @classmethod
-    async def from_ctx(cls, ctx, ignore_guild: bool = False):
+    async def from_ctx(cls, ctx, ignore_guild: bool = False, ignore_channel: bool = False, raise_error: bool = True):
```

Comment:
Should use a try-except statement instead of disabling the error, the rest of the function is not safe to run if active_character is None

Reply:
Fixed, though it isn't as graceful as I wanted since I'll need each individual call to be try-except'ed, because each call might fail with NoCharacter and I wouldn't want the subsequent calls for the next character to not pass.